src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -86,8 +88,21 @@ namespace AppInstaller::CLI::Workflow
             }
 #endif
 
-            auto factory = ConfigurationRemoting::CreateOutOfProcessFactory();
-            Configuration::SetModulePath(context, factory);
+            IConfigurationSetProcessorFactory factory;
+
+            // Since downgrading is not currently supported, only use dynamic if not running as admin.
+            if (Settings::ExperimentalFeature::IsEnabled(Settings::ExperimentalFeature::Feature::ConfigureSelfElevation) &&

```

Comment:
I added an experimental feature `configureSelfElevate` to allow easier testing of this from the command line.